tests/test_log.py

```diff
@@ -3,14 +3,22 @@
 import sys
 import unittest
 import yaml
-import json
 from io import StringIO
 
 from mock import Mock, patch
 from patroni.config import Config
 from patroni.log import PatroniLogger
 from queue import Queue, Full
 
+try:
+    from pythonjsonlogger import jsonlogger
+    import json
```

Comment:
are we also checking for a possible(?) json module import error here? or only import when `json_formatter_is_available = True`? if latter, then maybe move it to the end of the "try" block?